onnxscript/tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -447,6 +460,14 @@ def sample_inputs_col2im(op_info, device, dtype, requires_grad, **kwargs):
         sample_inputs_func=sample_inputs_native_group_norm,
         supports_out=False,
     ),
+    opinfo_core.OpInfo(
+        "max_pool1d",
+        variant_test_name="empty_strides",
+        op=torch.ops.aten.max_pool1d,
+        aten_name="max_pool1d",
+        dtypes=common_dtype.floating_types_and(torch.bfloat16),
```

Comment:
Done

Reply:
Reverted. I got an error message: torch.max_pool1d doesn't support torch.half. And I also check OPS_DB, and found that there is no torch.hald tests inside. I think it's simply not supported in torch.

Reply:
LMK what you think.

Reply:
Specifically, they only support it on CUDA: https://github.com/pytorch/pytorch/blob/9331b7fa05d92c76e0325a848c50e2756ce6cfaa/torch/testing/_internal/common_methods_invocations.py#L12866-L12867

I wonder maybe other cases having the same issue that we don't know if it's tested with float16, even though we allow it in ops_test_data.py.

Reply:
Yes, just like I said in my email about "The float16 precision checking for the onnx-script functions", about 20% of the function has no test case of torch.float16 defined, this need further investigation.
So that's OK for this PR. Thanks!